core/api/src/services/mongoose/wallets.ts

```diff
@@ -146,6 +167,7 @@ export const WalletsRepository = (): IWalletsRepository => {
 
   return {
     findById,
+    findForAccountById,
```

Comment:
Similar optional nit here
```suggestion
    findByAccountIdAndId,
```